alertmanager/config/config.go

```diff
@@ -113,7 +113,7 @@ func (c *Config) RemoveReceiverFromRoute(receiver string) {
 	for i, route := range c.Route.Routes {
 		c.Route.Routes[i] = removeReceiverFromRouteImpl(receiver, route)
 	}
-	prunedRoutes := funk.Filter(c.Route.Routes, func(x *Route) bool {return x != nil})
+	prunedRoutes := funk.Filter(c.Route.Routes, func(x *Route) bool { return x != nil })
```

Comment:
This should be linted / auto formatted.  If you committed it incorrectly, should a test have failed?

Reply:
I guess this isn't a lint that is actually checked by gofmt or the linter. Personal preference?